board.py

```diff
@@ -96,7 +96,7 @@ def disp_board(self):
         for row in range(len(self.board[i])):
             print(row+1, end= ' ')
             
-        
+    # does not print a newline at the end
```

Comment:
When you disp_board then print something immediately after, the next print shows up on the same line as your columns.